### PR TITLE
small issues addressed in #4817 fixed

### PR DESCRIPTION
**Pull request by 12xx12**
the original pull request was merged before i could commit those changes

### CODE REVIEW

File: src/Generating/FinishGen.cpp [hunk 1]
```diff
@@ -1783,15 +1783,7 @@ void cFinishGenOreNests::GenerateOre(
 		int BaseY = nestRnd % a_MaxHeight;
 		nestRnd /= a_MaxHeight;
 		// if the NestSize is smaller then four this breaks
```
Comment (bearbin) [inline src/Generating/FinishGen.cpp:1785]: Maybe this comment is not quite up to date anymore.

Comment (12xx12) [reply src/Generating/FinishGen.cpp:1785]: maybe...

Comment (12xx12) [reply src/Generating/FinishGen.cpp]: changed

**Comment by bearbin** [comment]
Thanks, sorry for merging the old PR too early. I blame tiger :P

**Comment by tigerw** [comment]
Σ(っ °Д °;)っ

**Pull request merged**